.github/actions/setup-env/action.yaml

```diff
@@ -0,0 +1,25 @@
+name: Setup Environment
+description: 'Setup Environment for workflows'
+
+inputs:
+  go:
+    description: 'Install Golang'
+    required: false
+    default: 'false'
+  go-cache:
+    description: 'Cache Golang dependencies'
+    required: false
+    default: 'true'
+  go-version-file:
+    description: 'Path to go.mod to pull go version from'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      if: ${{ inputs.go == 'true' }}
```

Comment:
Forgot to mention last week, but since all the jobs in our workflows always install go we could do away with this optional install flag

Reply:
Done